plugins/node/instrumentation-fs/src/instrumentation.ts

```diff
@@ -172,14 +195,18 @@ export default class FsInstrumentation extends InstrumentationBase<FS> {
         span.end();
       }
     };
+    Object.assign(rv, original);
+
+    return rv;
   }
 
   protected _patchCallbackFunction<T extends (...args: any[]) => ReturnType<T>>(
```

Comment:
I see you didn't modify `_patchPromiseFunction` accordingly. Is it because the two 2 level functions don't have a promise signature `fs.promises.realpath.native`?

Can you please add a comment to describe this behavior, or modify the code so it's consistent in all cases?